src/main/java/com/linkedin/kmf/services/ConsumeService.java

```diff
@@ -224,9 +223,7 @@ public boolean isRunning() {
     private final Sensor _recordsDelay;
     private final Sensor _recordsDelayed;
 
-    public ConsumeMetrics(Metrics metrics, final Map<String, String> tags) {
-      this.metrics = metrics;
-
+    public ConsumeMetrics(final Metrics metrics, final Map<String, String> tags) {
```

Comment:
Why add final here and not everywhere else it could be added?  IMO final for parameters degrades readability.  If reassigning to parameters is a code quality problem then perhaps a better check style file would be the way to go.

Reply:
It seems that the latest Kafka still use Java 1.7 for source compatibility. And if compilation with Java 1.7 will fail if we do not add `final` here. `final` is only needed here because it is used to measure `consume-availability-avg` in this method.